### ISSUE TITLE
[Bug]: AttributeError: module 'pydantic.v1' has no attribute 'json'. Did you mean: 'Json'?

### ISSUE THREAD

**Issue by ryanpeach** [run-llama/llama_index#12537]
### Bug Description

```
/usr/local/lib/python3.10/site-packages/llama_index_client/core/jsonable_encoder.py:39: in <module>
    encoders_by_class_tuples = generate_encoders_by_class_tuples(pydantic.json.ENCODERS_BY_TYPE)
E   AttributeError: module 'pydantic.v1' has no attribute 'json'. Did you mean: 'Json'?
```

Interestingly enough I cant find this in the main branch?

### Version

0.10.24.post1

### Steps to Reproduce

I think you just need to `import llama_index_client` but idk where that package lives.

**Comment by logan-markewich** [comment]
Just saw this in a PR I made -- trying to debug what the issue could possibly be to cause this (the import / tests works fine locally 😓)

**Comment by ryanpeach** [comment]
I traced it to here `from llama_index.core.ingestion.api_utils import get_client`

**Comment by logan-markewich** [comment]
@ryanpeach its the client for llama index cloud and llama-parse -- its in a private repo at the moment

**Comment by logan-markewich** [comment]
Since the client is auto-generated with a tool called fern, not so easy to fix. Will make an issue on their repo though. 

In the meantime, I can fix this for our CICD by requiring pydantic==2.x for some integration packages

**Comment by logan-markewich** [comment]
@ryanpeach I can't put that in the reqs, as not every integration supports v2 yet

**Comment by logan-markewich** [comment]
tbh I think I need to fix this in the client 😢

**Comment by logan-markewich** [comment]
Fixed -- `pip install llamaindex-py-client>=0.1.16` -- will update reqs in another PR

**Comment by francipvb** [comment]
Hello all,

To simplify this and avoid a break later, add a constraint to pydantic>=1.10.0<1.10.15 for the pydantic 1 dependency.

The `v1` module was added in the 1.10.15 version.
